test/lit.cfg

```diff
@@ -472,6 +472,7 @@ config.substitutions.insert(0,
 # Set available features we allow tests to conditionalize on.
 if platform.system() != 'Windows':
     config.available_features.add('crash-recovery')
+    config.available_features.add('shell')
```

Comment:
Hmm, I thought that lit added it itself.  In any case, this should not be in this case, but its own as:

```
if not config.use_lit_shell:
  config.available_features.add('shell')
```

We can use the lit shell in other cases as well if we need/wanted to.